addons/l10n_it_edi_withholding/models/account_chart_template.py

```diff
@@ -14,9 +14,13 @@ def _get_it_withholding_account_account(self):
 
     @template('it', 'account.tax')
     def _get_it_withholding_account_tax(self):
-        additionnal = self._parse_csv('it', 'account.tax', module='l10n_it_edi_withholding')
-        self._deref_account_tags('it', additionnal)
-        return additionnal
+        additional = self._parse_csv('it', 'account.tax', module='l10n_it_edi_withholding')
+        self._deref_account_tags('it', additional)
+        return additional
+
+    @template('it', 'account.fiscal.position')
+    def _get_it_withholding_account_fiscal_position(self):
+        return self._parse_csv('it', 'account.fiscal.position', module='l10n_it_edi_withholding')
```

Comment:
What? Where is it?

Reply:
Who? 👇🏻  Where? 👈🏻  When? ☝🏻 Whaddaf* 👉🏻 😅  

Sorry, this was part of an unfinished work to create fiscal position for withholding taxes that ended up in the PR. 

I removed the file from the commit. 🤣